sandbox.go

```diff
@@ -220,13 +220,11 @@ func (sb *sandbox) delete(force bool) error {
 			continue
 		}
 
-		if !force {
```

Comment:
Just wondering, this was never done then?

Reply:
ping @abhi did you check on this?